viewing_party/main.py

```diff
@@ -0,0 +1,135 @@
+
+
+def create_movie(title, genre, rating):
+    movie_dictionary = {
+        "title": title, 
+        "genre": genre, 
+        "rating": rating
+        } 
+    if not title == None and not genre == None and not rating == None:
+        return movie_dictionary
+    else:
+        return None 
```

Comment:
Nice Libby! I recommend you adding creation of the movie_dictionary into the if statement just so that if None values are passed, you aren't creating a dictionary and THEN returning None.